src/workerd/io/worker.h

```diff
@@ -572,6 +580,11 @@ class Worker::Api {
   virtual void setModuleFallbackCallback(kj::Function<ModuleFallbackCallback>&& callback) const {
     // By default does nothing.
   }
+
+  // Return the virtual file system for this worker, if any.
+  virtual kj::Maybe<const VirtualFileSystem&> getVirtualFileSystem() const {
```

Comment:
If the former question regarding the VFS being a maybe is accepted this can be changed to not return a maybe either and just be a pure virtual function.